loc/en.js

```diff
@@ -363,9 +372,9 @@ module.exports = {
       type: 'Transaction type',
       watchOnlyError: 'Watch only wallets cannot send transactions',
     },
-    warning: 'Warning: ',
+    warning: 'Warning:',
     warningGeneral:
-      'Please be aware that in the process of using the Secure Transaction feature, a part of the funds left in your wallet might be blocked. This is a normal procedure linked to UTXO and the blockchain parameters of the Bitcoin Vault wallet. Your funds will be unblocked once the transaction is verified (after approximately 24 hrs) or canceled (within 24 hrs).',
+      'Warning: Please be aware that in the process of using the Secure Transaction feature, a part of the funds left in your wallet might be blocked. This is a normal procedure linked to UTXO and the blockchain parameters of the Bitcoin Vault wallet. Your funds will be unblocked once the transaction is verified (after approximately 24 hrs) or canceled (within 24 hrs).',
```

Comment:
recently we deleted "Warning" from here

Reply:
pls check

Reply:
I can confirm we removed the word "Warning: " from line 377 in another PR recently. Have you pulled the latest develop?

Reply:
done